packages/server/src/usecases/message_create.ts

```diff
@@ -7,17 +7,35 @@ export async function messageCreate(
 ): Promise<Message> {
   const { roomName, content } = messageCreate;
   const room = db.data.rooms.find((room) => room.name === roomName);
+  const userColors = db.data.userColors;
   const id = new Date().getTime();
 
+  if (!userColors[authorName]) {
+    userColors[authorName] = getRandomColor();
+  }
   const message = {
     id,
     content,
     roomName,
     authorName,
+    color: userColors[authorName],
   };
   room.messages.push(message);
 
   await db.write();
 
   return message;
 }
+
+function getRandomColor(): string {
+  const colors = [
+    "#e74c3c", // red
+    "#3498db", // blue
+    "#2ecc71", // green
+    "#f1c40f", // yellow
+    "#9b59b6", // purple
+    "#1abc9c", // turquoise
+    "#e67e22", // orange
+  ];
+  return colors[Math.floor(Math.random() * colors.length)];
```

Comment:
```suggestion
  return colors[Math.floor(Math.random() * colors.length)];
```
export pick จาก utils มาแล้วใช้อันนั้นแทน